### PR TITLE
Rename to dutch

### PR DESCRIPTION
**Pull request by KarafiziArtur**
https://github.com/local-motion/product/issues/342

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -39,28 +39,28 @@ class App extends React.Component {
...
-                        {/* <Route exact path="/workspace/:initiativeId" key="WorkspacePage" component={Workspace}/>
-                        <Route exact path="/workspace/:initiativeId/login" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
+                        {/* <Route exact path="/actie/:initiativeName" key="WorkspacePage" component={Workspace}/>
+                        <Route exact path="/actie/:initiativeName/inloggen" key="WorkspaceLogin" render={ props => <Workspace {...props} signInHandler={this.signInHandler}/> } /> */}
                         
-                        <Route exact path="/workspace/login" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
-                        <Route exact path="/workspace/add-find-playground" key="WorkspaceAddFindPlayground" component={Workspace}/>
+                        <Route exact path="/actie/inloggen" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/zoeken" key="WorkspaceAddFindPlayground" component={Workspace}/>
```
Comment (maxvisser) [inline src/App.jsx]: After seeing this implemented, I would change url to /actie/starten instead of zoeken to align it with h1

Comment (KarafiziArtur) [reply src/App.jsx]: Done!

File: src/App.jsx [hunk 2]
```diff
@@ -39,28 +39,28 @@ class App extends React.Component {
...
                         
-                        <Route exact path="/workspace/login" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
-                        <Route exact path="/workspace/add-find-playground" key="WorkspaceAddFindPlayground" component={Workspace}/>
+                        <Route exact path="/actie/inloggen" key="WorkspaceLogin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/starten" key="WorkspaceAddFindPlayground" component={Workspace}/>
 
-                        <Route exact path="/workspace/:initiativeId/join" key="WorkspaceJoin" component={WorkspaceJoin}/>
-                        <Route exact path="/workspace/:initiativeId/login" key="WorkspaceJoin" render={props => <Workspace {...props} signInHandler={this.signInHandler}/>}/>
+                        <Route exact path="/actie/:initiativeName/join" key="WorkspaceJoin" component={WorkspaceJoin}/>
```
Comment (maxvisser) [inline src/App.jsx]: join = aansluiten

**Comment by maxvisser** [comment]
saw one thing after approval

File: src/App.jsx (hunk 2 above)
Comment (KarafiziArtur) [reply src/App.jsx]: But if it needs for SEO purposes so ok, I'll rename it.

Comment (KarafiziArtur) [reply src/App.jsx]: Done

**Comment by dimitriirtimid** [comment]
merged

**Comment by dimitriirtimid** [comment]
After discussion with @KarafiziArtur and @maxvisser I removed the id from the playground name as used in the url. url encoding of the playground is not simply replace the space with underscore. Also the set of character allowed to used in a playgrounds name is reduced and the encoding is tested with all allowed characters.
@maxvisser please review issue https://github.com/local-motion/product/issues/400
closing this PR

**Pull request closed without merging**